framework/source/class/qx/event/handler/Mouse.js

```diff
@@ -375,6 +377,17 @@ qx.Class.define("qx.event.handler.Mouse",
 
       this.__fireEvent(domEvent, type, target);
 
+      /*
+       * In order to normalize middle button click events we
+       * need to fire an artificial click event if the client
+       * fires auxclick events for non primary buttons instead.
+       * 
+       * See https://github.com/qooxdoo/qooxdoo/issues/9268
+       */
+      if (type == "auxclick" && domEvent.button == 1) {
+        this.__fireEvent(domEvent, "click", target);
```

Comment:
A new property, maybe "middleButtonProducesClick" that determines whether your simulated "click" event is generated when the native "auxclick" occurs. The new property would default to true, to do exactly what you have there right now, but if it is set to false, then an "auxclick" would not generate a simulated "click".